<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grafy3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="QWidget" name="page">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>300</width>
            <height>239</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Nowy graf</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="GraphSizeWidget" name="newGraphProvider" native="true"/>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>300</width>
            <height>239</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Macierz połączeń</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="1">
                <widget class="MatrixView" name="matrixWidget" native="true"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>M=</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>300</width>
            <height>239</height>
           </rect>
          </property>
          <attribute name="label">
           <string>ConsistentGraphGenerator</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Ilość wierzchołków:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="consistentVertexCount"/>
             </item>
             <item>
              <widget class="QPushButton" name="consistentGraphGeneratorTrigget">
               <property name="text">
                <string>generuj</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>300</width>
            <height>239</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Dijkstra</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="DijkstraView" name="dijkstraView" native="true"/>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>300</width>
            <height>239</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Centrum</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="CenterView" name="centrumView" native="true"/>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>300</width>
            <height>239</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Macierz odległości</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>D = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DistanceMatrix" name="distanceMatrix" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_7">
          <attribute name="label">
           <string>Minimalne drzewo rozpinajace</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="removeTrigget">
               <property name="text">
                <string>usuń</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="MSTTrigger">
               <property name="text">
                <string>wyznacz</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="GraphPainter" name="graphPainter" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphPainter</class>
   <extends>QWidget</extends>
   <header>graphpainter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphSizeWidget</class>
   <extends>QWidget</extends>
   <header>graphsizewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MatrixView</class>
   <extends>QWidget</extends>
   <header>matrixview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DijkstraView</class>
   <extends>QWidget</extends>
   <header>dijkstraview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CenterView</class>
   <extends>QWidget</extends>
   <header>centerview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DistanceMatrix</class>
   <extends>QWidget</extends>
   <header>distancematrix.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
